specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-11-01/managedClusters.json

```diff
@@ -1619,6 +1619,19 @@
           "default": "172.17.0.1/16",
           "description": "A CIDR notation IP range assigned to the Docker bridge network. It must not overlap with any Subnet IP ranges or the Kubernetes service address range."
         },
+        "OutboundType": {
```

Comment:
Adding these properties classifies as a breaking api change and a new api-version is required:
https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Breaking%20changes%20guidelines.md#new-property-added-to-response

Reply:
@ryansbenson this is a new api-version. We are just a few weeks late on releasing `2019-11-01`

Reply:
I see, this isn't a merge to master.